src/Polar/kernels.jl

```diff
@@ -1,6 +1,7 @@
 import KernelAbstractions: @index
+using CUDA
+using ForwardDiff
```

Comment:
Do we need to import `ForwardDiff` explicitly here? I think `ForwardDiff` is already present in this scope

Reply:
Done